package.json

```diff
@@ -46,6 +46,7 @@
         "discord.js": "discordjs/discord.js",
         "dotenv": "^8.2.0",
         "ms": "^2.1.2",
+        "pascal-case": "^2.0.1",
```

Comment:
do we really need a new dep?

Reply:
Don't merge for now, I'll rewrite it proper later then :)

Reply:
It's gone :)